src/hotspot/share/gc/shenandoah/shenandoahConcurrentGC.cpp

```diff
@@ -525,6 +526,8 @@ void ShenandoahConcurrentGC::op_init_mark() {
 
   heap->set_concurrent_mark_in_progress(true);
 
+  _mark.start_mark();
```

Comment:
Style: call `start_mark()` here maybe? Matches the use of `end_mark()` elsewhere.

Reply:
Okay. I am award of asymmetry, but not sure this is better.